extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -16,6 +16,13 @@
 import {BaseCarousel} from './base-carousel';
 import {Layout} from '../../../src/layout';
 
+/** @const */
+const DIR_FORWARD = 1;
```

Comment:
to me this makes the code a little better to read - let me know if you feel too strongly about this, i can replace.